modules/client-configuration/pages/clients-oracle.adoc

```diff
@@ -0,0 +1,127 @@
+[[clients-oracle]]
+= Registering {oracle} Clients
+
+This section contains information about registering traditional and Salt clients running {oracle} operating systems.
+
+[IMPORTANT]
+====
+You are responsible for arranging access to {oracle} base media repositories and {oracle} installation media.
+====
+
+ifeval::[{suma-content} == true]
+
+endif::[]
+
+
+[IMPORTANT]
+====
+Traditional clients are not available on {oracle}{nbsp}8.
+{oracle}{nbsp}8 clients are only supported as Salt clients.
+====
+
+
+== Server Requirements
+
+Before you begin, check that your {productname} Server meets the requirements at xref:installation:hardware-requirements.adoc[].
+
+Taskomatic uses one CPU core, and requires at least 3072{nbsp}MB of RAM.
+To ensure that taskomatic has access to enough memory, open the [path]``/etc/rhn/rhn.conf`` configuration file, and add this line:
+
+----
+taskomatic.java.maxmemory=3072
+----
+
+Restart Taskomatic:
+----
+systemctl restart taskomatic
+----
+
+
+
+== Channel and Repository Management
+
+
+
+
+ifeval::[{suma-content} == true]
+.Procedure: Adding Client Tools Channels
+. On the {productname} Server, add the appropriate {oracle} channels:
++
+* For {oracle} 6:
++
+From the {webui}, add [systemitem]``Oracle Linux 6 x86_64``.
++
+From the command prompt, add [systemitem]``oraclelinux6-x86_64``.
++
+* For {oracle} 7:
++
+From the {webui}, add [systemitem]``Oracle Linux 7 x86_64``.
++
+From the command prompt, add [systemitem]``oraclelinux7-x86_64``.
++
+* For {oracle} 8:
++
+From the {webui}, add [systemitem]``Oracle Linux 8 x86_64``.
```

Comment:
I wonder if this will also mark all the children repos automatically. This needs to be tested when Kevin has sumatoolbox ready.

Reply:
It should mark all the mandatory children channels automatically. See https://github.com/uyuni-project/uyuni/pull/1941